pkgs/tools/misc/x11idle/default.nix

```diff
@@ -1,12 +1,14 @@
 { stdenv, xlibs, fetchgit, libXScrnSaver, libX11 }:
 
-stdenv.mkDerivation {
-  name = "x11idle-unstable-2018-06-16";
+stdenv.mkDerivation rec {
+
+  version = "9.1.9";
+  name = "x11idle-org-${version}";
 
   src = fetchgit {
     url = "https://code.orgmode.org/bzg/org-mode.git";
```

Comment:
Would it be possible to jut use https://code.orgmode.org/bzg/org-mode/raw/release_9.1.13/contrib/scripts/x11idle.c as the source? That would be much smaller.